tensorflow/python/ops/image_ops_impl.py

```diff
@@ -4083,6 +4083,19 @@ def extract_glimpse_v2(
   * If the coordinates are not normalized they are interpreted as
     numbers of pixels.
 
+  Usage Example:
+    ```python
+    BATCH_SIZE = 1
+    IMAGE_HEIGHT = 3
+    IMAGE_WIDTH = 3
+    CHANNELS = 1
+    GLIMPSE_SIZE = (2, 2)
+    image = tf.reshape(tf.range(9, delta=1, dtype=tf.float32),
```

Comment:
Also I would love to change these ones to the new style but first, I couldn't understand what is going on here and secondly, I couldn't think of a way to manually type this variable like x=[[...]] thing since I couldn't understand it as I said.

Reply:
Ok actually fixed some of them but there is one last more remaining. This is the fixes: https://github.com/tensorflow/tensorflow/pull/35388/commits/0a89b29136c8233d521bf26450c8492fa72576a1